crates/wasmi/src/store.rs

```diff
@@ -67,6 +68,22 @@ impl StoreIdx {
 /// A stored entity.
 pub type Stored<Idx> = GuardedEntity<StoreIdx, Idx>;
 
+pub struct ResourceLimiterRef<'a>(pub(crate) Option<&'a mut (dyn crate::ResourceLimiter)>);
```

Comment:
After seeing many instances of
```rust
ResourceLimiterRef(match &mut store.limiter {
    Some(q) => Some(q.0(&mut store.data)),
    None => None,
})
```
I think it is fair to say that convenience getter would be nice. With the following 2 getters it is possible to get rid of all those `ResourceLimiterRef(match { .. })` constructs in the rest of the codebase and hide internals as well in `impl Store`:
```rust
pub(crate) fn store_inner_and_resource_limiter_ref(&mut self) -> (&mut StoreInner, ResourceLimiterRef) {
    let resource_limiter = ResourceLimiterRef(match &mut self.limiter {
        Some(q) => Some(q.0(&mut self.data)),
        None => None
    });
    (&mut self.inner, resource_limiter)
}

pub(crate) fn resource_limiter_ref(&mut self) -> ResourceLimiterRef {
    self.store_inner_and_resource_limiter_ref().1
}
```
This allows us to remove some `pub(crate)` annotations.

Reply:
Ok.